wurst/_items/HuntingNet.wurst

```diff
@@ -76,18 +76,16 @@ function onHuntNet()
     let target = GetSpellTargetUnit()
     let pos = target.getPos()
     real NET_DURATION = 0.
-    if target.isTroll()
-        NET_DURATION = HERO_DURATION
-    else 
-        NET_DURATION = NORMAL_DURATION
+    NET_DURATION = (target.isTroll())? HERO_DURATION : NORMAL_DURATION
```

Comment:
You don't need extra parenthesis.

`NET_DURATION = target.isTroll() ? HERO_DURATION : NORMAL_DURATION `.